tests/test_utils.py

```diff
@@ -30,6 +33,50 @@ def test_not_exclusive(self) -> None:
         self.assertEqual(nodes.are_exclusive(xass1, xnames[1]), False)
         self.assertEqual(nodes.are_exclusive(xass1, xnames[2]), False)
 
+    @pytest.mark.skipif(not PY38_PLUS, reason="needs assignment expressions")
+    def test_not_exclusive_walrus_operator(self) -> None:
+        from astroid import extract_node, nodes
+
+        node_if, node_body, node_or_else = extract_node(
+            """
+        if val := True:  #@
+            print(val)  #@
+        else:
+            print(val)  #@
+        """
+        )
+        node_if: nodes.If
+        node_walrus = next(node_if.nodes_of_class(nodes.NamedExpr))
+
+        self.assertEqual(nodes.are_exclusive(node_walrus, node_if), False)
```

Comment:
Could you use `pytest` style `assert` instead of `unittest`. The tests are quite old, but new ones are preferably written in `pytest` style.

Reply:
Sure, but what do you mean by `pytest style assert`? is it a raw python `assert`?

Reply:
Yeah!